interfaces/acados_template/acados_template/c_templates_tera/acados_solver.in.c

```diff
@@ -260,12 +259,14 @@ int acados_create()
         nh[i]     = NH;
         nphi[i]   = NPHI;
         nr[i]     = NR;
```

Comment:
unrelated, but why is NRN not defined?

Reply:
I guess it never was, few lines above, I deleted:
`     int nr_e[N+1];`
but one can only really check using an example with `nr != nr_e`